specs/Standards/NonFungibleToken/Core.md

```diff
@@ -157,15 +157,15 @@ The following behavior is required, but contract authors may name this function
 // * `receiver_id`: the `receiver_id` argument given to `nft_transfer_call`
 // * `token_id`: the `token_id` argument given to `nft_transfer_call`
 // * `approved_token_ids`: if using Approval Management, contract MUST provide
-//   set of original approved accounts in this argument, and restore these
-//   approved accounts in case of revert.
+//   record of original approved accounts in this argument, and restore these
+//   approved accounts and their approval IDs in case of revert.
 //
 // Returns true if token was successfully transferred to `receiver_id`.
 function nft_resolve_transfer(
   owner_id: string,
```

Comment:
This is one line higher than I can easily make a commit suggestion, would you mind changing this line to be `owner__id`, as it currently says `sender_id` (in the comments)
https://github.com/near/NEPs/pull/303/files#diff-9132c1e8e04717bd55ad822800135f4094e4814a1d5fdb50e86d1204681dbf8eR156